### PR TITLE
Add S3 configuration instructions to FrequentTasks.md

### PR DESCRIPTION
**Pull request by lbakerchef**
Signed-off-by: Lincoln Baker <lbaker@chef.io>

### Description

Inward-facing developer documentation.

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: dev-docs/FrequentTasks.md [hunk 1]
```diff
@@ -331,3 +331,34 @@ knife ssl fetch
 knife ssl check
 ```
 
+## Using S3 instead of Bookshelf
+
+If you would like to configure chef-server to use S3 instead of Bookshelf, you can
+type the following into the bash shell of your chef-server.  This assumes you are
+using a root login.  Make obvious substitutions as needed, e.g. vip, external_url,
```
Comment (vinay-satish) [inline dev-docs/FrequentTasks.md:338]: The variables to be substituted are in chef-server.rb ?

**Review by vinay-satish — APPROVED** [review]
LGTM except for the one comment

File: dev-docs/FrequentTasks.md (hunk 1 above)
Comment (lbakerchef) [reply dev-docs/FrequentTasks.md:338]: Yes.  See bash comment below `# copies this into the bottom of /etc/opscode/chef-server.rb`

Comment (lbakerchef) [reply dev-docs/FrequentTasks.md:338]: It's a script you can copy into the bash shell of your running chef-server to make the changes you need to `chef-server.rb`, `environment`, etc.

**Review by jashaik — APPROVED** [review]

**Pull request merged**